testsuite/mixed-domain/src/test/java/org/jboss/as/test/integration/domain/mixed/eap640/DomainAdjuster640.java

```diff
@@ -188,6 +186,41 @@
     }
 
 
+    private List<ModelNode> adjustInfinispan(final PathAddress subsystem) throws Exception {
+        final List<ModelNode> list = new ArrayList<>();
+
+        list.add(getWriteAttributeOperation(subsystem.append("cache-container", "server").append("transport", "jgroups"), "stack", new ModelNode("udp")));
+
+        //Statistics need to be enabled for all cache containers and caches
+        list.add(setStatisticsEnabledTrue(
```

Comment:
Statistics-enabled were introduced in 6.4.0, so there is no need to set them to true for 6.4.0, only for 6.3.0